manifests/07_deployment.yaml

```diff
@@ -24,9 +24,17 @@ spec:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
     spec:
+      securityContext:
+        runAsNonRoot: false
```

Comment:
this should probably stay unset? You do actually want to run as NonRoot, if until now this was a `restricted` workload, this would make it `baseline`, no?

Reply:
oh dear, that is a type 🙈

Reply:
if it stays unset it will not comply to `restricted` as per https://github.com/kubernetes/kubernetes/blob/b36c927d957e585ce54d415be4d7e4621e189d85/staging/src/k8s.io/pod-security-admission/policy/check_runAsNonRoot.go#L114-L125

Reply:
I just verified in a local cluster, using this PR the `openshift-apiserver-operator` pod matches `openshift.io/scc: privileged`, in a cluster without this change it matches `openshift.io/scc: anyuid`

Reply:
binding to privileged will be gone once https://github.com/openshift/cluster-kube-apiserver-operator/pull/1338 merges